ansible/ansible-ai-connect-service
# PR to fix ansible-ai-connect-service: Cross-site Scripting (XSS) in serialize-javascript

Jira Issue: <https://issues.redhat.com/browse/AAP-31381>


## Description

PR to fix ansible-ai-connect-service: Cross-site Scripting (XSS) in serialize-javascript

## Testing

### Steps to test
1. Pull down the PR
2. Run `npm install` and node packages should install as expected w/o failures
3. Run `npm audit fix` to verify if there's any vulnerabilities, currently there's 0.

### Scenarios tested

NA

## Production deployment
